client/src/components/applicants/ApplicationStages.js

```diff
@@ -2,75 +2,57 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import Applicant from './Applicant'
+import { Droppable } from 'react-beautiful-dnd'
 
 import * as actions from '../../redux/actions/actions'
 
 export class ApplicationStages extends Component {
```

Comment:
Minor detail, but I think the name could be changed to ApplicationStage (singular)...?